### PR TITLE
PS-5025: Setting compression flags for fields on DD load

### PR DESCRIPTION
**Pull request by dutow**

### CODE REVIEW

File: mysql-test/t/percona_bug_ps5025.test [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# PS-5025: field flag's weren't correctly set for compressed columns, causing a crash on an insert after the column was altered
```
Comment (laurynas-biveinis) [inline mysql-test/t/percona_bug_ps5025.test]: Please merge with one of the existing compressed column testcases

File: storage/innobase/dict/dict0dd.cc [hunk 2]
```diff
@@ -2918,18 +2918,23 @@ static inline dict_table_t *dd_fill_dict_table(const Table *dd_tab,
 
     bool is_stored = innobase_is_s_fld(field);
 
+    ulint is_compressed =
```
Comment (laurynas-biveinis) [inline storage/innobase/dict/dict0dd.cc]: const

**Review by laurynas-biveinis — CHANGES_REQUESTED** [review]
Please use GCA starting at 5.6 to add the testcase to there and to 5.7 (please address the testcase merging comment too)

**Comment by dutow** [comment]
Updated.

**Review by laurynas-biveinis — APPROVED** [review]

**Pull request merged**